### DISCUSSION TITLE
About drug conversion

### DISCUSSION THREAD

**Discussion by guanchengcheng** [MIT-LCP/mimic-code#1647] [MIMIC-IV] [answered]
Hello, 
I have difficulty calculating the score by vasoactive drugs, the unit when calculating vasopressin in the VIS score is U/Kg/min, but the unit of vasopressin I extracted in imputevents is unit/hour, is U equal to unit here? Or unit here is another unit of measurement, if so, please let me know. 
Hope my question is clear enough, thank you.

**Comment by alistairewj** [comment, marked as answer]
For vasopressin U does mean unit.

You might want to look at the norepinephrine equivalent doses query, which
has some light data cleaning / harmonization for the vasopressors.

**Comment by guanchengcheng** [comment]
Thanks to your help, I think I have a new understanding of drug dose conversion.
